OpenRA.Mods.RA/Traits/LeavesHusk.cs

```diff
@@ -10,17 +10,38 @@
 
 using System.Linq;
 using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.Common.Warheads;
 using OpenRA.Primitives;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.RA.Traits
 {
+	public enum OwnerType { Victim, Killer, InternalName }
+
 	[Desc("Spawn another actor immediately upon death.")]
 	public class LeavesHuskInfo : ITraitInfo
 	{
 		[ActorReference, FieldLoader.Require]
+		[Desc("Actor to spawn on death.")]
 		public readonly string HuskActor = null;
 
+		[Desc("Probability the husk actor spawns.")]
+		public readonly int Probability = 100;
+
+		[Desc("Owner of the husk actor. Allowed keywords:" +
+			"'Victim', 'Killer' and 'InternalName'.")]
+		public readonly OwnerType OwnerType = OwnerType.Victim;
+
+		[Desc("Player to use when 'InternalName' is defined on 'OwnerType'.")]
```

Comment:
`InternalName` definitely needs some more explanation. Perhaps you can state here that it would probably be `World` or `Creeps`.